account_cutoff_accrual_picking/models/account_cutoff.py

```diff
@@ -89,8 +89,11 @@ def _prepare_line(self, line):
             taxes = fpos.map_tax(taxes)
             # The quantity received on PO line must be deducted of all moves
             # done after the cutoff date.
+            from_string = fields.Date.from_string
             moves_after = line.move_ids.filtered(
-                lambda r: r.state == 'done' and r.date > self.cutoff_date)
+                lambda r: r.state == 'done'
+                and from_string(r.date) > from_string(self.cutoff_date)
```

Comment:
this could use an inline comment explaining that one of the two fields is actually a datetime, not a date.